popup.html

```diff
@@ -96,6 +96,7 @@
     </div>
     <!-- End of container -->
 
+    <script src="./popup-script.js"></script>
```

Comment:
Why the `./popup-script.js`? You have not been consistent in your JavaScript path conventions. Also: why is this `<script>` tag not at the top? Or conversely, why aren't all the scripts included here at the bottom?

Reply:
I put the './popup-script.js' at the button because if I put it at the top, firebase doesn't recognize the div, therefore the sign in with google button UI is not show